packages/server/graphql/public/typeDefs/acceptRequestToJoinDomain.graphql

```diff
@@ -6,7 +6,7 @@ extend type Mutation {
     """
     DomainJoinRequest id
     """
-    requestId: ID!
+    requestId: Int!
```

Comment:
-1 We should keep this `ID!` as it's still an id. If we plan to use it in a connection, we should also make it globally unique by following the gqlId join/split pattern.